src/env/node/git/git.ts

```diff
@@ -100,6 +102,11 @@ export const GitErrors = {
 	tagConflict: /! \[rejected\].*\(would clobber existing tag\)/m,
 	unmergedFiles: /is not possible because you have unmerged files/i,
 	unstagedChanges: /You have unstaged changes/i,
+	unmergedChanges: /error:\s*you need to resolve your current index first/i,
+	ambiguousArgument: /fatal:\s*ambiguous argument ['"].+['"]: unknown revision or path not in the working tree/i,
+	entryNotUpToDate: /error:\s*Entry ['"].+['"] not uptodate\. Cannot merge\./i,
+	changesWouldBeOverwritten: /error:\s*Your local changes to the following files would be overwritten/i,
```

Comment:
This already exists above. You should remove one of them. I also suggest continuing the convention of alphabetizing these.